src/arm32le.h

```diff
@@ -42,27 +42,12 @@
 #define DES_BS_ASM			0
 #define DES_BS				1
 #if __ARM_NEON && !JOHN_NO_SIMD
-#if 0
-#define DES_BS_VECTOR			2
-#define DES_BS_ALGORITHM_NAME		"DES 64/64 NEON"
-#elif 0
-#define DES_BS_VECTOR			3
-#define DES_BS_VECTOR_SIZE		4
-#define DES_BS_ALGORITHM_NAME		"DES 64/64 NEON + 32/32"
-#elif 0
-#define DES_BS_2X64
-#define DES_BS_VECTOR			4
-#define DES_BS_ALGORITHM_NAME		"DES 64/64 X2 NEON"
-#elif 1
+#if 1
 #define DES_BS_VECTOR			4
 #define DES_BS_ALGORITHM_NAME		"DES 128/128 NEON"
-#elif 0
-#define DES_BS_VECTOR			5
-#define DES_BS_VECTOR_SIZE		8
-#define DES_BS_ALGORITHM_NAME		"DES 128/128 NEON + 32/32"
 #else
-#define DES_BS_VECTOR			8
-#define DES_BS_ALGORITHM_NAME		"DES 128/128 X2 NEON"
+#define DES_BS_VECTOR			2
+#define DES_BS_ALGORITHM_NAME		"DES 64/64 NEON"
```

Comment:
We should make similar edits to `arm64le.h`. That file doesn't exist in core (I should probably add it separately), so was missed by the merge.